canonical-web-and-design/ubuntu.com
# Image builder

## Done
This branch contains the entire Image builder feature.

## QA

QA at http://imagebuilder.staging.demo.haus/build

- Go to /build
- Sign in
- Select a board, OS and optionally add some snaps.
- Click Build
- You should see a thank you page
- You will **not** receive the completed email as the webhook is targeting production. (It's likely @nottrobin will get the email which he may kindly forward for your QA)
- Once you have the email click the links in the email and check they work

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2387
